### PR TITLE
#41 Auth Bugfix

### PR DESCRIPTION
**Pull request by BenBolthouse**
Pull request includes various code fixes to seeders, routes, validators and utility functions.
* Demo user login changed in auth view,
* Seeders include hashed passwords and sequence resets requiring reseeding of development and production environments,
* Logout bug rectified by putting keyword `return` ahead of the `res.render()` function in auth routes.

### CODE REVIEW

File: db/seeders/20210107223757-demo-spotcollection-seed.js [hunk 1]
```diff
@@ -1,37 +1,78 @@
...
+      {}
+    );
   },
-
   down: (queryInterface, Sequelize) => {
-
-    return queryInterface.bulkDelete('SpotCollections', {} , {});
-  }
+    queryInterface.sequelize.query(
+      'ALTER SEQUENCE "SpotCollections_id_seq" RESTART WITH 1;'
```
Comment (r-maloney) [inline db/seeders/20210107223757-demo-spotcollection-seed.js:74]: Very cool- didn't know you could do that

Comment (BenBolthouse) [reply db/seeders/20210107223757-demo-spotcollection-seed.js:74]: Nifty little trick I discovered in the Express solo project.

**Comment by BenBolthouse** [comment]
Changes were made to get auth up and running again, ready for review.

**Comment by r-maloney** [comment]
Everything working well after reseeding the database. Ship it!

**Review by r-maloney — APPROVED** [review]
Looking good!

**Pull request merged**